cortx-monitorQuickstartGuide.md

```diff
@@ -49,8 +49,8 @@ This guide provides a step-by-step walkthrough for getting you CORTX-Monitor Ser
 
         :page_with_curl: **Notes:** 
         
-      - If the https://raw.githubusercontent.com/Seagate/cortx-prvsnr/dev/cli/src/cortx-prereqs.sh?token=APA75GY34Y2F5DJSOKDCZAK7ITSZC link is not accessible, generate new one.
-      - Visit https://github.com/Seagate/cortx-prvsnr/blob/dev/cli/src/cortx-prereqs.sh and naviagte to RAW > Copy URL > Use the URL for deployment.
+      - If the https://raw.githubusercontent.com/Seagate/cortx-re/main/scripts/third-party-rpm/install-cortx-prereq.sh?token=APBA7EBWCTPTEOH5PPWFYYTBCN23Y link is not accessible, generate new one.
+      - Visit https://github.com/Seagate/cortx-re/blob/main/scripts/third-party-rpm/install-cortx-prereq.sh and naviagte to RAW > Copy URL > Use the URL for deployment.
```

Comment:
Why need token? Its dynamic and so token key in the link can be changed later. 
Above I have mentioned generic link. Please check that.

Reply:
agree, lets avoid token when not needed

Reply:
without token, we are getting 404 error. We have given a note to change is the token if it is not valid anymore.